app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx

```diff
@@ -199,3 +200,10 @@ export const ProjectSettingsPageContainer = () => {
     </SettingsLayout>
   );
 };
+
+export const CenteredContainerForPage = ({ children }) => (
```

Comment:
Today it is responsible for centered content, tomorrow the behavior can be different.
Please rename to `SettingsPageContent`, also rename corresponding `className`.
And also usually we don't hold more than one component in a single file in case them don't use each other.
You can move this component together with styles to the `content` folder.

Reply:
fixed ✅